data-runtime/src/main/java/io/micronaut/data/runtime/mapper/sql/SqlDTOMapper.java

```diff
@@ -27,6 +27,7 @@
  * @param <S> The source type.
  * @param <R> The result type
  */
+@Deprecated(forRemoval = true, since = "4.2")
```

Comment:
This is not being used anymore it seems so maybe we can deprecate it and remove later?